### ISSUE TITLE
No Option to Accept Game Start

### ISSUE THREAD

**Issue by spicy385** [ValveSoftware/Dota2-Gameplay#15926]
### Description

Each time I try to start a game, I wait for the load time, but I never get the option to "Accept" the game. I'm now sitting at a 30-minute ban, as it's been done a few times now.

### Screenshots

<img width="301" alt="Screen Shot 2024-03-12 at 7 58 55 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/163214789/10904bc0-a7ca-4db2-9624-c7b2ad42e0bf">

**Comment by ThisisSierra117** [comment]
I know a few people also experiencing this exact same thing, including myself with a 30 minute ban

**Comment by Goodies5150** [comment]
Yeah what happens after this, do we get our ban uplifted or do we just get penalized for no reason?

**Comment by commentator24** [comment]
#6567 upvote 👍 that
it's probably problem on your side not server.
screenshot console log to see what's happening.
someone also comment -tcp protocol on steam.exe might help
